sources/stylesheets/content-area/_text.scss

```diff
@@ -313,7 +313,8 @@
     font-weight: 700;
   }
 
-  .custom-btn {
+  .custom-btn,
+  .readmore-link {
```

Comment:
The selector and it rules should be defined in the [article block stylesheet partial](https://github.com/Voog/design-santiago/blob/master/sources/stylesheets/components/articles/_article.scss).